### PR TITLE
Legal (TOS and Copyright)

### PR DESCRIPTION
**Pull request by MegaApplePi**
- added `Legal/TOS` and `Legal/Copyright` folders
- added the `en.md` files
- copied-and-pasted the TOS and Copyright (DMCA) into the respective files
- fixed formatting to attempt to match original

### CODE REVIEW

**Comment by cl8n** [comment]
While on the topic, we should probably do something with the other old /p/ pages as well, like art, challengeart, doyoureallywanttoaskpeppy, inactive, twitter, amazon, bestof20yy, etc. since they likely won't see a mention on the new site

**Comment by MillhioreF** [comment]
As per discussion in the dev discord all completely static pages should be imported into the wiki format, and can be given special URLs if need be. Many of those pages aren't static and will have to be redesigned or removed, but some (twitter, askpeppy, maybe inactive?) can certainly be moved over.

**Comment by peppy** [comment]
yep, we want to move as much of them to the new wiki system as possible (anything that doesn't require dynamic content implementations, basically).

File: wiki/Legal/Copyright/en.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Designated Agent to Receive Notification of Claimed Infringement: 
+
+**Email**: [pe@ppy.sh](mailto:pe@ppy.sh) (24 hour response guaranteed) 
+
+\- or -
+
+**Postal Address**: 
+
+<pre>
```
Comment (cl8n) [inline wiki/Legal/Copyright/en.md]: use triple backticks (` ``` `) instead

File: wiki/Legal/TOS/en.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+**9\. PRIVACY POLICIES**
+
+osu.ppy.sh finds your privacy most important, and we value your use of our services.
+osu.ppy.sh's services are free of any charge, no prior registration is needed for the use of these services.
+osu.ppy.sh will explicitly ask for information about you when such information is needed for use of the services provided by osu.ppy.sh.
+Any submission of any personal information is done only with your voluntary act (website), or automatically (game client software) where necessary to provide diagnostics and feedback.
+osu.ppy.sh keeps all private and personal information private, and does not, or will not share this information with any third parties except under the following circumstances :
+
+When such action is necessary to conform to legal requirements or comply with legal process.
```
Comment (cl8n) [inline wiki/Legal/TOS/en.md]: probably just use a list here

File: wiki/Legal/Copyright/en.md (hunk 1 above)
Comment (MegaApplePi) [inline wiki/Legal/Copyright/en.md]: Using triple backticks enables the syntax highlighter (idk about how it'll appear on osu!wiki though).

File: wiki/Legal/TOS/en.md (hunk 2 above)
Comment (MegaApplePi) [inline wiki/Legal/TOS/en.md]: I am not sure if I should modify the tos any further then what was given.

File: wiki/Legal/Copyright/en.md (hunk 1 above)
Comment (cl8n) [inline wiki/Legal/Copyright/en.md]: if you don't specify a language it should have the same effect as `<pre>`

**Comment by MegaApplePi** [comment]
The new lines is just an appearance wise thing while editing it in the text. Markdown considers them as a single line.

File: wiki/Legal/Copyright/en.md (hunk 1 above)
Comment (MegaApplePi) [inline wiki/Legal/Copyright/en.md]: Whoops, one of the markdown previewers I used has displayed it with syntax highlighting .-.

I'll a push a change.

**Comment by cl8n** [comment]
That's a good point; I guess I just commented on it because it's different to most of the other articles.

This should be ready to merge now

**Pull request merged**